README.md

```diff
@@ -2,25 +2,25 @@
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat)](http://bioconda.github.io/recipes/odgi/README.html)
 
-## optimized dynamic genome/graph implementation
+## optimized dynamic genome/graph implementation (odgi)
 
-Representing large genomic [variation graphs](https://github.com/vgteam/vg) with minimal memory overhead requires a careful encoding of the graph entities.
-It is possible to build succinct, _static_ data structures to store queryable graphs, as in [xg](https://github.com/vgteam/xg), but dynamic data structures are more tricky to implement.
+`odgi` provides an efficient, succinct dynamic DNA sequence graph model, as well as a host of algorithms that allow the use of such graphs in bioinformatic analyses.
 
-`odgi` follows the dynamic [GBWT](https://github.com/jltsiren/gbwt) in developing a byte-packed version of the graph and paths through it.
-Each node is represented by a byte array into which we write variable length integers to represent, 1) the node sequence, 2) its edges, and 3) the paths crossing the node.
+Careful encoding of graph entities allows `odgi` to efficiently compute and transform [pangenomes](https://pangenome.github.io/) with minimal overheads.  `odgi` implements a dynamic data structure that can be updated on the fly. This contrasts with _static_ data structures, such as [xg](https://github.com/vgteam/xg).
 
-The edges and path steps are recorded relativistically, as deltas between the current node id and the target node id, where the node id corresponds to the rank in the global array of nodes.
-Graphs built from biological data sets tend to have local partial order, and when sorted the stored deltas will tend to be small.
-This allows them to be compressed with a variable length integer representation, resulting in a small in-memory footprint at the cost of packing and unpacking. 
+`odgi` implements dynamic [GBWT](https://github.com/jltsiren/gbwt) as a byte-packed version of the graph.
+Each node is represented by a `byte array` into which variable length integers are written to represent: 1) node sequences, 2) edges, and 3) paths through nodes.
```

Comment:
This is also not correct, we should sync an update I made to the docs that's in the paper draft

Reply:
Removed